backend/server.js

```diff
@@ -341,6 +325,7 @@ sbp('sbp/selectors/register', {
       await sbp('chelonia.db/delete', `_private_hidx=${cid}#0`)
       await sbp('chelonia.db/delete', `_private_keyop_idx_${cid}_0`)
       await sbp('chelonia.db/set', cid, '')
+      sbp('chelonia/private/removeImmediately', cid)
```

Comment:
Should this change be ported to #2737?

Reply:
No, this bugfix doesn't really affect #2737. This is to remove contract state from memory when they're deleted.